talk/owt/sdk/base/win/msdkvideoencoder.cc

```diff
@@ -174,6 +174,9 @@ int MSDKVideoEncoder::InitEncodeOnEncoderThread(
   // Init the encoding params:
   MSDK_ZERO_MEMORY(m_mfxEncParams);
   m_mfxEncParams.mfx.CodecId = codec_id;
+  if (codec_id == MFX_CODEC_AVC) {
+    m_mfxEncParams.mfx.CodecLevel = MFX_PROFILE_AVC_CONSTRAINED_BASELINE;
```

Comment:
Does profile-level-id in SDP correctly show this setting?

Reply:
yes CB is included in SDP. But profile higher Main/High encoded stream from windows, when decoded on ios/win. So currently the workaroud is to use CB only when publishing, unless further we allow setting profile when publishing.

Reply:
Shall we add comments here to mark it workaround currently?